libpod/runtime_ctr.go

```diff
@@ -770,7 +770,11 @@ func (r *Runtime) LookupContainer(idOrName string) (*Container, error) {
 func (r *Runtime) GetContainers(filters ...ContainerFilter) ([]*Container, error) {
 	r.lock.RLock()
 	defer r.lock.RUnlock()
+	return r.GetContainersWithoutLock(filters...)
+}
 
+// GetContainersWithoutLock is same as GetContainers but without lock
+func (r *Runtime) GetContainersWithoutLock(filters ...ContainerFilter) ([]*Container, error) {
```

Comment:
Note to self: The runtime lock is actually not really useful anymore - we can probably just remove it. No action needed in this PR, I'll make a fresh one for it.